test/uk/gov/hmrc/taxhistory/services/EmploymentServiceSpec.scala

```diff
@@ -412,10 +411,11 @@ class EmploymentServiceSpec extends PlaySpec with MockitoSugar with TestUtil{
       val npsEmployment3 = NpsEmployment(randomNino.toString(),3,"offNo3","ref3","empname3",None,false,false,LocalDate.now(),None, false, EmploymentStatus.Live)
       val npsEmployments = List(npsEmployment1,npsEmployment2,npsEmployment3)
       val rtiData = RtiData("QQ0000002", rtiEmployments)
-      val rtiDataHelper = new RtiDataHelper(rtiData)
-
-      val onlyRtiEmployments = rtiDataHelper.onlyInRTI(npsEmployments)
-      onlyRtiEmployments.size mustBe 2
+//      val auditable = new AuditableTestImpl
```

Comment:
If it's not being used, remove this commented code please

Reply:
Reinstated code